jujugui/static/gui/src/app/components/accordion-section/accordion-section.js

```diff
@@ -0,0 +1,97 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2017 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+class AccordionSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: this.props.startOpen || !this.props.title
+    };
+  }
+
+  _toggle() {
+    this.setState({open: !this.state.open});
+  }
+
+  _getHeight() {
```

Comment:
Docstring. I am confused about this method as it can return a boolean or a number. Is there a semantic difference? Can we avoid this inconsistency?